assets/js/base/components/radio-control/style.scss

```diff
@@ -51,28 +49,28 @@
 .wc-block-components-radio-control {
 	.wc-block-components-radio-control__input {
 		appearance: none;
-		background: transparent;
+		background: #fff;
 		border: 2px solid currentColor;
 		border-radius: 50%;
 		display: inline-block;
 		height: 1.25rem;
```

Comment:
In this file, every time we update the `min-height`/`min-width`, we should update the `height` and `width` values as well. In this case, they should be in relative units so they adapt to the themes or any a11y preference from the user.

I think it should be safe using the `em()` function here. So in this selector, we would need to make these changes:
```diff
.wc-block-components-radio-control__input {
	...
-	height: 1.25rem;
- 	width: 1.25rem;
+	height: em(24px);
+	width: em(24px);
}
```

Same for the other sizes which are modified in this file.